v1/proto/service/gribi.proto

```diff
@@ -50,7 +50,10 @@ service gRIBI {
 // ModifyRequest is a message used by the client to manipulate the state of
 // the RIB on the target device.
 message ModifyRequest {
-  // A group of requests to add/modify/remove a single AFT entry
+  // A group of requests to add/modify/remove a single AFT entry.
+  // gRIBI server relies on clients being conservative in what they send.
```

Comment:
This statement seems a bit fragile to me. Is there something specific that we're calling out the clients being conservative about?

Reply:
ummm...make sense. I should keep the wording succint here, and leave reasoning in other docs like the motivation.md